src/DeserializationPojo/ListUsersAdPojo.java

```diff
@@ -1,7 +1,6 @@
-package pojo2;
+package DeserializationPojo;
 
-public class add2 {
-	
+public final class ListUsersAdPojo {
```

Comment:
I think the `Ad` data returned for both list users and list resource is the same. So you can create a single pojo class and use it in both places. In that case it makes sense to use the name `Ad` or `AdPojo` without any api prefix. 

Usually the class name should represent what the object is. Use additional prefix/suffix only if more context is required to explain what the object it.